mattlianje/gregarious-wolf
# Pitch Extraction REVAMPED

- Now using the time intervals from the `labelled` table
- Should not be weird on the final edge cases

**TESTING**
We want to test locally, so for a random video (ideally kind of short just for testing) do the following. DJ Trump Video: "_https://www.youtube.com/watch?v=JZRXESV3R74_"

- Turn on localhost and everything
- Create "labelled" table locally if you haven't
- Add a "pitch" column to "labelled" table
- `cd import-clips`
- `npm start https://www.youtube.com/watch?v=JZRXESV3R74 log`

This should fill the local table with empty rows for all time intervals.

- `cd audio-analysis`
- `python sendPitch.py log`
- Enter the DJ Link above

When finished running, the "pitch" column for that URL in "labelled" table should be filled out NICELY now.
